Amplify/Categories/Auth/Models/AuthCodeDeliveryDetails.swift

```diff
@@ -19,7 +19,8 @@ public struct AuthCodeDeliveryDetails {
     public let attributeKey: AuthUserAttributeKey?
 
     public init(destination: DeliveryDestination,
-                attributeKey: AuthUserAttributeKey? = nil) {
+                attributeKey: AuthUserAttributeKey? = nil)
```

Comment:
I recall we agreed on a different convention for these cases: To do a line break before the first parameter, and another one for the last parenthesis.

i.e.
```swift
    public init(
        destination: DeliveryDestination,
        attributeKey: AuthUserAttributeKey?  = nil
    ) {
```

This comment applies to all function definitions that have been changed.